lib/js/components/SelectList/SelectListItemToggle/SelectListItemToggle.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+	<select-list-item
+		class="selectListItemToggle"
+		:icon-left="icon"
+		:state="isLoading ? SELECT_LIST_ITEM_STATES.LOADING : SELECT_LIST_ITEM_STATES.DEFAULT"
+		:label="label"
+		:size="size"
+	/>
+</template>
+
+<script lang="ts">
+import { SELECT_LIST_ITEM_STATES, SELECT_LIST_SIZES } from '../SelectList.consts';
+import { IconItem, ICONS } from '../../Icons/Icon';
+import SelectListItem from '../SelectListItem/SelectListItem.vue';
+
+export default {
+	name: 'SelectListItemToggle',
+	components: {
+		SelectListItem,
+	},
+	props: {
+		iconOff: {
+			type: Object,
+			default: null,
+			validator(icon) {
```

Comment:
does default `null` pass this validation?

Reply:
Good question. I see no warnings, so I assume default value doesn't go through the validator.

Reply:
what if you pass `null` directly as prop value?

Reply:
Still nothing.